process/subprocesses/ghsci.py

```diff
@@ -1055,7 +1049,7 @@ def drop(self, table=''):
     def generate_report(
         self,
         language: str = 'English',
-        report: str = 'indicators',
+        report='indicators',

```

Comment:
in general, it is good practice to specify the type of variables.  I would not make this change, unless you are intending to refactor to accept types other than string...